jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/TooltipTests.java

```diff
@@ -0,0 +1,163 @@
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static org.testng.Assert.*;
+import static org.testng.AssertJUnit.assertFalse;
+
+public class TooltipTests extends TestsInit {
+
+    private static String textOnBottomTooltip = "Tooltip on bottom";
+    private static String textOnTopTooltip = "Tooltip on top";
+    private static String textOnRightTooltip = "Tooltip on right";
+    private static String textOnLeftTooltip = "Tooltip on left";
+    private static String textOnHTMLTooltip = "<em>Tooltip</em> <u>with</u> <b>HTML</b>";
+    private static String textOnDisabledButton = "Disabled tooltip";
+
+    private static String bottom = "bottom";
+    private static String top = "top";
+    private static String left = "left";
+    private static String right = "right";
+
+    private static String topButtonText = "Tooltip on top";
+    private static String bottomButtonText = "Tooltip on bottom";
+    private static String rightButtonText = "Tooltip on right";
+    private static String leftButtonText = "Tooltip on left";
+    private static String htmlButtonText  = "Tooltip with HTML";
+    private static String disabledButtonText = "Disabled button";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @DataProvider(name = "TooltipsPlacements")
+    public static Object[][] tooltipsPlacement() {
+        return new Object[][]{
+                {tooltipOnTopButton, top},
+                {tooltipOnBottomButton, bottom},
+                {tooltipOnLeftButton, left},
+                {tooltipOnRightButton, right}
+        };
+    }
+
+    @DataProvider(name = "Tooltips")
+    public static Object[][] tooltips() {
+        return new Object[][]{
+                {tooltipOnTopButton, textOnTopTooltip},
+                {tooltipOnBottomButton, textOnBottomTooltip},
+                {tooltipOnLeftButton, textOnLeftTooltip},
+                {tooltipOnRightButton, textOnRightTooltip},
+                {tooltipWithHTML, textOnHTMLTooltip},
+                {wrapperForDisabledButton, textOnDisabledButton}
+        };
+    }
+
+    @DataProvider(name = "Buttons")
+    public static Object[][] buttonsText() {
+        return new Object[][]{
+                {tooltipOnTopButton, topButtonText},
+                {tooltipOnBottomButton, bottomButtonText},
+                {tooltipOnLeftButton, leftButtonText},
+                {tooltipOnRightButton, rightButtonText},
+                {tooltipOnBottomButton, bottomButtonText},
+                {tooltipWithHTML, htmlButtonText}
+        };
+    }
+
+    @Test(dataProvider = "Tooltips")
+    public void getTooltipTextTests(Tooltip tooltip, String tooltipText) {
+        assertEquals(tooltip.getTooltipText(), tooltipText);
+    }
+
+    @Test(dataProvider = "Tooltips")
+    public void tooltipIsInvisibleTests(Tooltip tooltip, String __) {
+        assertFalse(tooltip.isVisible());
```

Comment:
we should use construction like:
`tooltip.is().hidden()`
so could you please try to implement that approach? I think should be a way to realize that